introduction.md

```diff
@@ -3,3 +3,8 @@ Hi All!
 It's me :-)
 
 My favorite CodeScool path is JavaScript
+
+Deadly skills:  
+* Git
+* JavaScript
+* Ruby
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?